Client/Client.py

```diff
@@ -25,15 +27,15 @@ def __init__(self):
 
     def receive_message(self, message):
         print("Server sent: ", message)
-        message_split = message.split(";")
-        if message_split[0] == "kakao":
-            print("I sent: csoki")
-            self.send_message("csoki")
+        if message["type"] == client_message_constants.MessageType.CONN:
```

Comment:
Server shall send server message constants, and client shall send client message contstants in my opinion. Maybe the other way around. But furely not both both. Then there is no point of splitting, use simply message_constants...